agent/stats/task_linux_test.go

```diff
@@ -21,13 +21,12 @@ import (
 	"testing"
 	"time"
 
-	mock_nswrapper "github.com/aws/amazon-ecs-agent/agent/utils/nswrapper/mocks"
-
 	apicontainer "github.com/aws/amazon-ecs-agent/agent/api/container"
 	apitask "github.com/aws/amazon-ecs-agent/agent/api/task"
 	apitaskstatus "github.com/aws/amazon-ecs-agent/agent/api/task/status"
 	mock_netlink "github.com/aws/amazon-ecs-agent/agent/eni/netlinkwrapper/mocks"
 	mock_resolver "github.com/aws/amazon-ecs-agent/agent/stats/resolver/mock"
+	mock_nswrapper "github.com/aws/amazon-ecs-agent/agent/utils/nswrapper/mocks"
```

Comment:
NOTE: I tried to alphabetize these (ie put `mock_nswrapper` before `mock_resolver`, but `go fmt` switches them back to this.  I'll defer to `go fmt` -- it's probably sorting by path.

Reply:
did you try goimports? I believe that's what we're using?

Reply:
I ran `make gogenerate` and `make goimports` then `go fmt ./agent/...` 
this is what came out the other end of that ^^